tests/integration_tests/databases/api_tests.py

```diff
@@ -1563,34 +1563,6 @@ def test_get_select_star_not_allowed(self):
         rv = self.client.get(uri)
         self.assertEqual(rv.status_code, 404)
 
-    def test_get_select_star_datasource_access(self):
```

Comment:
Non-blocking comment: would be good if you could mention why these tests were removed.

Reply:
These tests were poorly written and relied on weird side effects.

Reply:
thank you!